lib/ansible/plugins/inventory/__init__.py

```diff
@@ -160,6 +161,22 @@ def __init__(self):
         self.inventory = None
         self.display = display
 
+    def get_option(self, option):
+
+        value = super(BaseInventoryPlugin, self).get_option(option)
+
+        if is_template(value):
```

Comment:
Will this template different types? templar.is_template seems to handle other types as well as call this function.

Reply:
should handle strings directly and complex types recursively

Reply:
ah, nvmd, its changed ....